### DISCUSSION TITLE
Omit the current working directory in tree view

### DISCUSSION THREAD

**Discussion by yttriumz** [eza-community/eza#1051] [Q&A, Howto, Help]
`eza -lF --no-quotes -a -g --time-style iso -T --level 2` is the command I often use, which will give the following output:

```text
drwxr-xr-x - az az 07-08 16:18 ./
drwxr-xr-x - az az 07-08 16:18 ├── bar/
drwxr-xr-x - az az 07-08 16:18 │  ├── 1/
drwxr-xr-x - az az 07-08 16:18 │  ├── 2/
drwxr-xr-x - az az 07-08 16:18 │  └── 3/
drwxr-xr-x - az az 07-08 16:19 └── foo/
drwxr-xr-x - az az 07-08 16:19    ├── a/
drwxr-xr-x - az az 07-08 16:19    ├── b/
drwxr-xr-x - az az 07-08 16:19    └── c/
```

Is it possible to omit the `.` directory, so that the result would look like the following?

```text
drwxr-xr-x - az az 07-08 16:18 bar/
drwxr-xr-x - az az 07-08 16:18 ├── 1/
drwxr-xr-x - az az 07-08 16:18 ├── 2/
drwxr-xr-x - az az 07-08 16:18 └── 3/
drwxr-xr-x - az az 07-08 16:19 foo/
drwxr-xr-x - az az 07-08 16:19 ├── a/
drwxr-xr-x - az az 07-08 16:19 ├── b/
drwxr-xr-x - az az 07-08 16:19 └── c/
```

**Comment by gierens** [comment]
Hi @Alecton4 

When not specifying a target then the tree view will always start at `.`, but you can run your command against `*` to get:
![Screenshot from 2024-07-08 13-39-18](https://github.com/eza-community/eza/assets/49617392/04bacc29-c1f3-4ff0-b884-c880eb20b36b)

In case you are also interested in hidden items you need both `-a` and `.*` as additional target:
![Screenshot from 2024-07-08 13-39-39](https://github.com/eza-community/eza/assets/49617392/e60ee75e-d299-4f2f-bfd2-e5a866bb2e88)

Hope this answers the question. In case you feel like there should be an argument for this, feel free to open a feature request.
